scss/partials/_window.scss

```diff
@@ -801,3 +801,38 @@
     padding-top: 42px;
   }
 }
+
+.jsxc-custom-menu {
+  background: var(--jsxc-color-background-dark);
+  border: 1px solid var(--jsxc-color-border);
+  border-radius: 5px;
+  color: var(--jsxc-color-main-text);
+  display: none;
+  font-family: sans-serif;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  z-index: 1000;
+}
+
+.jsxc-custom-menu li {
```

Comment:
We should taking advantage from the sass nesting possibilities here too.

Reply:
if i move the li into the class, will the li then inherit from the class css fields?

Reply:
No, it just "inherits" the selector.

```
a {
  // foo

  b {
    // bar
  }
}

// will be transformed to

a {
  // foo
}

a b {
  // bar
}
```

Reply:
ok changed the css